packages/insured-bridge-relayer/src/index.ts

```diff
@@ -22,6 +22,10 @@ import { createBridgeAdapter } from "./canonical-bridge-adapters/CreateBridgeAda
 import { RelayerConfig } from "./RelayerConfig";
 config();
 
+function isErrorOutput<T>(input: PromiseSettledResult<T>): input is PromiseRejectedResult {
+  return "reason" in input;
+}
```

Comment:
this is really interesting

Reply:
Yeah we should add this to some general library and use it elsewehere

Reply:
Any suggestions as to where this should go?

Reply:
if we find we are needing this logic in more places id throw it in the sdk/utility file